src/app.module.ts

```
@@ -24,6 +23,8 @@ import { Comment } from "./feed/entities/comment.entity"
 import { Job } from "./jobs/entities/job.entity"
```

ESLint

[error] Replace `"./jobs/entities/job.entity"` with `'./jobs/entities/job.entity';`

(prettier/prettier)